drydock/hooks.py

```diff
@@ -0,0 +1,15 @@
+"""
```

Comment:
It's important to add documentation about this feature. Clarify that we moved from defining the init jobs statically to getting and writing them declaratively using a Tutor filter. I think that explaining the approach we took to define sync waves is also valuable. Last but not least, explain the usage of the `DRYDOCK_MIGRATE_FROM` variable.